src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/allelespecific/AS_StrandBiasTest.java

```diff
@@ -201,18 +195,6 @@ protected void parseRawDataString(ReducibleAnnotationData<List<Integer>> myData)
         myData.setAttributeMap(perAlleleValues);
     }
 
-
-    @SuppressWarnings({"unchecked", "rawtypes"})//FIXME
-    public void calculateRawData(final VariantContext vc,
```

Comment:
I thought calculateRawData was part of my ReducibleAnnotation interface?  Apparently not. I rather like the consistency, but you're the owner of this code now, so it's up to you.

Reply:
It's no longer part of the interface, I removed it because outside of a hack-y access in from the HaplotypeCaller in GATK4 it wasn't actually being called EXCEPT as part of annotateRawData() somewhere in the structure of annotations, thus it didn't seem necessary to expose it as part of the interface

Reply:
I removed it in the combine GVCFs branch because the calculateRawData argument was never getting used outside of the annotation framework itself (except in GATK4 as a hack to get around the reducible framework not having been added) so it seemed unnecessary to expose it as part of the external interface.